tests/test_http_parser.py

```diff
@@ -694,6 +694,7 @@ def test_http_response_parser_bad(response: Any) -> None:
         response.feed_data(b"HTT/1\r\n\r\n")
 
 
+@pytest.mark.skipif(not NO_EXTENSIONS, reason="Behaviour has changed in C parser")
```

Comment:
Better xfail instead..

Reply:
Or.. Can we include the minimum change needed to process the behavior difference?

Reply:
Could do, but I was planning to come back to this and figure out if we should make the behaviour consistent between parsers later. If there's a test here that explicitly allows it, then it seems like something that was wanted, so ideally we can tweak llhttp later to allow the behaviour again. So, I'd suggest leaving it like this for the moment, and I'll come back to it later.

Reply:
Okay, agreed. And in the terms of synchronizing the behaviors of C- and Python-based HTTP parsers, we should really invest into wiring Hypothesis.